### PR TITLE
Feature/cgf 179/tao health check

### PR DESCRIPTION
**Pull request by hutnikau**
Tao health check not just checks that Tao is installed but also that all the post install scripts were executed and installation was finished and Tao is ready to be used.

### CODE REVIEW

File: actions/class.Health.php [hunk 1]
```diff
@@ -10,6 +15,19 @@ class tao_actions_Health extends tao_actions_CommonModule
      */
     public function index()
     {
-        return;
+        try {
+            /** @var ApplicationService $applicationService */
+            $applicationService = $this->getServiceLocator()->get(ApplicationService::SERVICE_ID);
+            if ($applicationService->isInstallationFinished()) {
+                $this->setSuccessJsonResponse([
+                    'success' => $applicationService->isInstallationFinished()
```
Comment (Alroniks) [inline actions/class.Health.php:23]: It is enough just to set `true` here, as the first condition was successful.

File: actions/class.Health.php [hunk 2]
```diff
@@ -10,6 +15,19 @@ class tao_actions_Health extends tao_actions_CommonModule
...
+                ]);
+            } else {
+                throw new common_exception_Error('Installation is not finished');
+            }
+        } catch (Throwable $exception) {
+            $this->logError(sprintf('Error getting media relation: %s, ', $exception->getMessage()));
```
Comment (Alroniks) [inline actions/class.Health.php:29]: The message for a log seems weird. It is not about media.

**Review by Alroniks — COMMENTED** [review]
Here are some issues which require attention.

**Comment by hutnikau** [comment]
@Alroniks thanks for review. I pushed changes according to your comments. Please review it again

**Review by AndreyNegrov — APPROVED** [review]
Approved.

- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Pull request title and description are meaningful

**Review by Alroniks — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is `develop`

**Pull request merged**